Add what's expected of the Solutions Engineer on support (#67)
giantswarm/handbook: content/docs/support-and-ops/support/support-FAQ.md

- Did anything change in the cluster recently?
- Is the issue a real issue, or is it just a question about how to do something or a feature request? If it is not an issue, we can move the discussion to a GitHub issue.

## What's expected of the on-support engineer?

> Out of a Solutions Engineer on support, it is expected that:

Reviewer: ```suggestion
A Solutions Engineer on support shall:
```

> - the person qualifies the request by following the [Support Template](#support-template-qualifying-a-request)

Reviewer: ```suggestion
- qualify the request by following the [Support Template](#support-template-qualifying-a-request)
```

> - the person tries to perform some basic debugging according to opsrecipes/FAQ entries (look at pods logs, check if nodes are Ready, ...)

Reviewer: ```suggestion
- try to perform basic debugging according to opsrecipes/FAQ entries (look at pods logs, check if nodes are Ready, ...)
```

> - the person drives communication with the customer. This will allow us to have a more diverse face to the customer and not always the same people solving problems even though they might have helped in the background.

Reviewer: And so on...
